model/k8s/metrics.yaml

```diff
@@ -63,6 +63,28 @@ groups:
       - ref: network.interface.name
       - ref: network.io.direction
 
+  # k8s.container.* metrics
+  - id: metric.k8s.container.status.state
+    type: metric
+    metric_name: k8s.container.status.state
+    stability: experimental
+    brief: ""
+    instrument: updowncounter
+    unit: "{container}"
+    attributes:
+      - ref: k8s.container.status.state
+        requirement_level: required
+  - id: metric.k8s.container.status.reason
+    type: metric
+    metric_name: k8s.container.status.reason
+    stability: experimental
+    brief: ""
+    instrument: updowncounter
```

Comment:
What's the reason for this being an updowncounter?

Also, this seems like a string-valued metric here.

I see other comments to this effect, but the gist is that these types of metrics have very subtle edges when you use them in practice, leading to surprising issues.  THe prometheus "avoid missing metrics" is an example.

While, longer term, this type of data is something we absolutely want in the Entities signal, we should probably put some caveats on "state as metric" concerns that you'll run into with these.

Reply:
Alright, the logic makes sense for default aggregation.

And if i understand your example, will EVERY possible reason be reported every time interval? If so this makes a lot more sense.  That wasn't fully clear to me reading the description.

Reply:
Maybe add this explanation as part of `note`?

Reply:
Looks good to me. @jsuereth please also review.

Reply:
Looks good!